man/trollolo.1.md

```diff
@@ -36,6 +36,13 @@ and cards and has functionality for extracting data for burndown charts.
 
 ## COMMANDS
 
+### help -- Display help
+
+`trollolo help [COMMAND]`
+
+Display help and exit. Can take a command as an argument to display detailed
```

Comment:
I don't think it is needed saying that the command exit :wink: and it should be display**s** . What aboud this:

_Displays help about the available trollolo commands. Can take a command as an argument to display detailed information about it._